app/models/membership.rb

```diff
@@ -78,13 +78,20 @@ def archive!
   end
 
   def unarchive!
+    destroy_newbie_post
```

Comment:
I don't think we need this here since only active members can be archived (if not, then it should be fixed).

Reply:
When archived members apply to rejoin the group, they get newbie status so the group leader can see which members are waiting to be unarchived. So we need to destroy_newbie_post if  it's not nil. Or we can find a different approach for this.

Reply:
You're right, except they get the DEFAULT_POST_ID (which corresponds to 'feldolgozás alatt'. So I'd rename this method to `destroy_default_post`.